src/Model.java

```diff
@@ -58,6 +58,10 @@ public void runSimulationSequence() {
     this.cacheSnapshot = this.sim.getCacheSnapshot();
   }
 
+  public void setCalculator() {
+    this.calculator = new Calculator (this.cacheAccessTime, this.mmAccessTime, this.hitCount, this.missCount, this.isLoadThrough, this.blockSize);
```

Comment:
paset din nung output into the variables in Model using the getters in Calculator para just one call lang na ikaw in control, reference the runSimulationSequence I wrote c: